tests/system/python/e2e/test_e2e_coap_PI.py

```diff
@@ -44,6 +44,18 @@ def get_statistics_map(foglamp_url):
     jdoc = json.loads(r)
     return utils.serialize_stats_map(jdoc)
 
+def get_asset_tracking_details(foglamp_url, event=None):
```

Comment:
We should consider to move this and def like ping to move to utils or common module, so that we don't need to put this in every test module.